cdc/sink/mq/mq.go

```diff
@@ -386,10 +386,6 @@ func NewKafkaSaramaSink(ctx context.Context, sinkURI *url.URL,
 		return nil, cerror.WrapError(cerror.ErrKafkaInvalidConfig, err)
 	}
 
-	if err := replicaConfig.ApplyProtocol(sinkURI).Validate(); err != nil {
```

Comment:
why did you remove the validation?

Reply:
Move this to: https://github.com/pingcap/tiflow/pull/6038/files#diff-1aa9ba3321cb0edb368a32b5084da34322b0f9d6a600019beb724ab63bc5e071R162